ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedgerSpec.scala

```diff
@@ -93,22 +93,23 @@ class SqlLedgerSpec
       for {
         ledger <- createSqlLedger()
       } yield {
-        def allocateParty(displayName: String): Unit = {
+        def listPackages(displayName: String): Unit = {
```

Comment:
Doesn't look like `displayName` is used any more.

Reply:
Addressed in 165899128.